app/jenkins-for-jira-ui/src/api/redirectFromGetStarted.ts

```diff
@@ -0,0 +1,28 @@
+import { invoke, router } from '@forge/bridge';
+
+interface Context {
+	siteUrl: string;
+	appId: string;
+	environmentId: string;
+	moduleKey: string;
+}
+
+const redirectFromGetStarted = async (): Promise<string> => {
+	const context: Context = await invoke('redirectFromGetStarted');
+	const {
+		siteUrl,
+		appId,
+		environmentId,
+		moduleKey
+	} = context;
+
+	if (moduleKey === 'get-started-page') {
```

Comment:
A complete URL for the admin page looks something like `<jirahost>/jira/settings/apps/get-started/<appId>/<environmentId>/`. There's no way to get get the values in between the host and the path (which here is just /) via the frontend (React/window) so I've had to pass it in from the request context defined in the backend (via Forge). This is needed to make sure we don't render any of the elements on the landing page when we hit get-started.